README.md

```diff
@@ -41,6 +41,9 @@ module "rds" {
 | `deletion_protection` | bool | If the DB instance should have deletion protection enabled. The database can't be deleted when this value is set to true | true |
 | `skip_final_snapshot` | boolean | If set to true, no final snapshot of the database will be made when its deleted. | false |
 | `cloudwatch_logs_exports` | list(string) | List of log types to enable for exporting to CloudWatch logs. Each engine has different valid values | ['audit', 'error', 'general', 'slowquery'] |
+| `backup_retention_period` | number | The days to retain backups for. Must be between 0 and 35. Must be greater than 0 if the database is used as a source for a Read Replica. | 7 |
+| `backup_window` | string | The daily time range (in UTC) during which automated backups are created if they are enabled. Example: "09:46-10:16". Must not overlap with maintenance_window. | 07:01-07:31 (this is either midnight or 1am Mountain Time, depending on daylight savings) |
```

Comment:
Maybe add the format of this variable like you defined in maintenance_window below?

Reply:
I just stole these descriptions from the Terraform docs here: https://www.terraform.io/docs/providers/aws/r/db_instance.html#backup_retention_period.

Reply:
Done